scripts/install_custom_dc.sh

```diff
@@ -83,8 +83,29 @@ terraform init && terraform apply \
   -var="project_id=$PROJECT_ID" \
   -var="dc_website_domain=$DOMAIN" -auto-approve
 
-echo "Successfully launched the installer in $PROJECT_ID."
-echo "Please don't forget to email custom-datacommons-support@google.com for data access."
-echo "Please also make sure to click on the activation email for the newly created domain."
-echo "You should expect the instance to be accessible via https://$PROJECT_ID-datacommons.com"
-echo "within 30 minutes or so after custom-datacommons-support@google.com responds."
+_success_msg="
+###############################################################################
+# Status: Successfully launched the installer in $PROJECT_ID.
+###############################################################################
+
+###############################################################################
+# Action required:
+###############################################################################
```

Comment:
move line93 after line95?

move line99 after line102

and make one extra line between the two blocks?

Reply:
How does it look now?

Reply:
LG!